docs/pages/database-access/guides/rds.mdx

```diff
@@ -14,6 +14,11 @@ This guide will help you to:
   writing so it can't be used with Database Access.
 </Admonition>
 
+<Admonition type="note" title="MariaDB supported versions">
+  The minimum supported version of MariaDB is 10.6. Older versions don't support
```

Comment:
Nit: I would clarify that this is specifically for RDS MariaDB. Might be clear enough as it is given that this is RDS specific guide but I wonder if somebody might get confused by it anyway.